importlib_resources/readers.py

```diff
@@ -83,12 +83,28 @@ def is_file(self):
         return False
 
     def joinpath(self, *descendants):
-        try:
-            return super().joinpath(*descendants)
-        except abc.TraversalError:
-            # One of the paths did not resolve (a directory does not exist).
-            # Just return something that will not exist.
```

Comment:
It looks like this comment is lost in the rewrite.

Reply:
Yes, indeed. It should go in between lines 105,106.